### PR TITLE
Add new volunteer Anjali  Pant

### PR DESCRIPTION
**Pull request by Anjali-DA**

### CODE REVIEW

File: data/users/Anjali-DA.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anjali Pant",
+  "bio": "Data analyst",
+  "image": "https://twitter.com/AnjaliPant/photo",
```
Comment (codewithvoid) [inline data/users/Anjali-DA.json:4]: use this image url
https://pbs.twimg.com/profile_images/1586668940780240896/eQpl4gr3_400x400.jpg

File: data/users/Anjali-DA.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anjali Pant",
+  "bio": "Data analyst",
```
Comment (codewithvoid) [inline data/users/Anjali-DA.json:3]: nit: Data Analyst

File: data/users/Anjali-DA.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "Anjali Pant",
-  "bio": "Data analyst",
-  "image": "https://twitter.com/AnjaliPant/photo",
+  "bio": "Data Analyst",
+  "image": "https://pbs.twimg.com/profile_images/1586668940780240896/eQpl4gr3_400x400.jpg",
   "categories": [
     {
-      "slug": "data-science",
+      "slug": "Data-Science",
```
Comment (RakeshSangem) [inline data/users/Anjali-DA.json:7]: use this for slug `"slug": "data-science"`

You can also copy slug from `public/categories.json`

**Review by RakeshSangem — CHANGES_REQUESTED** [review]

File: data/users/Anjali-DA.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anjali Pant",
+  "bio": "Data Analyst",
+  "image": "https://pbs.twimg.com/profile_images/1586668940780240896/eQpl4gr3_400x400.jpg",
+  "categories": [
+    {
+      "slug": "Data-Science",
+      "topics": [
+        "Python",
```
Comment (codewithvoid) [inline data/users/Anjali-DA.json:9]: this can be kept lowercase. Update was just needed in bio

**Comment by codewithvoid** [comment]
Perfect. Thanks Anjali!

**Review by codewithvoid — APPROVED** [review]

**Review by RakeshSangem — APPROVED** [review]
LGTM🎉

**Pull request merged**